apps/vs-code-designer/src/main.ts

```diff
@@ -56,7 +56,13 @@ export async function activate(context: vscode.ExtensionContext) {
 
     runPostWorkflowCreateStepsFromCache();
 
-    await downloadExtensionBundle(activateContext);
+    try {
+      await downloadExtensionBundle(activateContext);
+    } catch (error) {
+      // log the error message the VSCode window and to telemetry.
+      const errorMessage = `Error downloading and extracting the Logic Apps Standard Extension Bundle: ${error.message}`;
```

Comment:
- Can you localize the text?
- Can you log the error in telemetry?
- I feel like the `Extension Bundle` shouldn't be uppercase

Reply:
fixed extension bundle..

the more I think about it the more I like just logging to telemetry instead of to the console, so I swapped it around. Do we localize telemetry messages?

Reply:
No, telemetry is not localized